docs/source/releases.rst

```diff
@@ -1,6 +1,68 @@
-=================
- Release History
-=================
+Release History
+###############
+
+
+v1.14.0 (2022-06-30)
```

Comment:
Does this need to hit v2.0.0 because of the removal of the `Device` container?

Reply:
I'd be fine with tagging as-is or doing (1), just want to make sure this isn't a slip up

Reply:
For us, I'm hoping to side-step this entirely as removal of a container or differing definitions in development/production environments makes this tough for us to fully support: https://github.com/pcdshub/device_config/pull/29

For any downstream users (?) that might have `Device`-based items, I think (1) would be appropriate to follow with semantic versioning deprecation. I'll push this back to draft until we get that resolved.